augurpy/cv.py

```diff
@@ -0,0 +1,27 @@
+from typing import List, Union
+
+from pandas import DataFrame
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.linear_model import LogisticRegression
+
+
+def run_cross_validate(
```

Comment:
Maybe we should rename this.
```suggestion
def run_cross_validation(
```

Reply:
I was thinking as well, I don't want to name it cross_validate because thats the skirt learn function I want to use in there - maybe _cross_validate?

Reply:
But sklearn only uses one of the two:

`run_cross_validation`
`cross_validation`

no? I'd use the one that is still available.

Reply:
Nevermind, you already fixed it!